orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -535,41 +536,42 @@ def test_forward_call_positional_and_keyword_arguments():
     prediction.backward()
 
 
+_ONE = torch.FloatTensor([1])
```

Comment:
is it possible to make the input local to the test and be passed in as an argument to the lambda?